### DISCUSSION TITLE
Is there yet a way to delete an account?

### DISCUSSION THREAD

**Discussion by diligiant** [Coho-For-Mastodon/Coho#72] [General]
Tried to delete/reinstall but I still have "Loading…" account??

**Comment by jgw96** [comment]
@diligiant could you explain more please? You mean when you delete and reinstall the PWA from the browser? Also, when you say you still have a "Loading..." account, could you explain more please?

**Reply by diligiant** [reply]
I uninstall the app, reinstall it from the Play Store and get =>
![image](https://user-images.githubusercontent.com/496822/227717344-424267d0-8c30-4523-8fc3-135ad7a54625.png)

**Comment by jgw96** [comment]
@diligiant ahh, you found a bug then actually! To work around this, clear the storage and cache for the app and then re-open the app
